cypress/support/commands.js

```diff
@@ -22,4 +22,19 @@
 //
 //
 // -- This will overwrite an existing command --
-// Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
\ No newline at end of file
+// Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
+Cypress.Commands.add('login', (username, password) => {
```

Comment:
```suggestion
Cypress.Commands.add('login', (username, password) => {
```
do not add the code, which you don't use. Remove it